BUG: Vet error, undefined err

**Is this a BUG REPORT or FEATURE REQUEST?**:
BUG
**What happened**:
```
# github.com/keikoproj/addon-manager/controllers
controllers/addon_controller.go:140:2: undefined: err
controllers/addon_controller.go:141:5: undefined: err
controllers/addon_controller.go:144:15: undefined: err
# github.com/keikoproj/addon-manager/controllers
vet: controllers/addon_controller.go:140:2: undeclared name: err
make: *** [vet] Error 2
```
**What you expected to happen**:
Build should work
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Addon Manager version
- Kubernetes version :
```
$ kubectl version -o yaml
```

**Other debugging information (if applicable)**:
- Addon status:
```
$ kubectl describe addon <addon-name>
```
- controller logs:
```
$ kubectl logs <addon-manager-pod>
```

--- a/cmd/manager/main.go
+++ b/cmd/manager/main.go
@@ -26,7 +26,6 @@ import (
 	addonmgrv1alpha1 "github.com/keikoproj/addon-manager/api/v1alpha1"
 	"github.com/keikoproj/addon-manager/controllers"
 	"github.com/keikoproj/addon-manager/pkg/version"
-
 	// +kubebuilder:scaffold:imports
 )
 

--- a/controllers/addon_controller.go
+++ b/controllers/addon_controller.go
@@ -123,7 +123,7 @@ func (r *AddonReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 		return reconcile.Result{}, ignoreNotFound(err)
 	}
-	
+
 	return r.execAddon(ctx, req, log, instance)
 }
 
@@ -137,7 +137,7 @@ func (r *AddonReconciler) execAddon(ctx context.Context, req reconcile.Request, 
 	ret, procErr := r.processAddon(ctx, req, log, instance)
 
 	// Always update status, cache
-	err = r.updateAddonStatus(ctx, instance)
+	err := r.updateAddonStatus(ctx, instance)
 	if err != nil {
 		// Force retry when status fails to update
 		ret.Requeue = true

